desk/app/logs.hoon

```diff
@@ -24,7 +24,9 @@
       def  ~(. (default-agent this %|) bowl)
       cor  ~(. +> [bowl ~])
   ::
-  ++  on-init  on-init:def
+  ++  on-init
+    ^-  (quip card _this)
+    `this(posthog `%info)
```

Comment:
Oops, somehow I missed this. Thanks!

Reply:
I would not find out, if I did not run the unit tests - they all now work. Just sayin' ;-)